place_tracker/README.md

```diff
@@ -4,7 +4,20 @@ A sample place tracking app that uses the [google_maps_flutter](https://github.c
 
 ## Getting Started
 
-For help setting up the plugin, see the plugin's [README](https://github.com/flutter/plugins/tree/master/packages/google_maps_flutter) page.
+To run this sample app, you will need an API key.
```

Comment:
This is all great. Since you're in here editing, though, this file needs some of the other things we typically have in our sample READMEs:

- Intro (brief description of what the app does)
  - For now, you should point out in giant bold letters that the app doesn't work on iOS yet and needs the master channel to run.
- Goals (what people can expect to learn/see in the sample)
  - For this app, it could be "Learn how to show and control a Google Map," "See how to place a marker on the map," etc.
- Important bits (what files people should look at first)
  - I'm guessing that would be place_map and place_details.
- Links to resources (copy these straight out of our other READMEs)

Have a look at the README for our Cupertino sample to see what I mean:

https://github.com/flutter/samples/blob/master/veggieseasons/README.md

Reply:
Done. Please let me know if you want me to add anything else.